burn-tensor/src/tests/stats/basic.rs

```diff
@@ -13,4 +13,58 @@ mod tests {
         let data_expected = Data::from([[2.4892], [15.3333]]);
         data_expected.assert_approx_eq(&data_actual, 3);
     }
+
+    #[test]
+    fn test_display_2d_tensor() {
+        let data = Data::from([[1, 2, 3], [4, 5, 6], [7, 8, 9]]);
+        let tensor: burn_tensor::Tensor<TestBackend, 2, burn_tensor::Int> = Tensor::from_data(data);
+
+        let mut output = format!("{}", tensor);
+        let mut expected = "Tensor {\n  data: [[1, 2, 3], [4, 5, 6], [7, 8, 9]],\n  shape:   [3, 3],\n  device:  Cpu,\n  backend: \"ndarray\",\n  dtype:   \"int\",\n}";
+        assert_eq!(output, expected);
+
+        let data_float = Data::from([[1.1, 2.2, 3.3], [4.4, 5.5, 6.6], [7.7, 8.8, 9.9]]);
+        let tensor_float: burn_tensor::Tensor<TestBackend, 2, burn_tensor::Float> =
+            Tensor::from_data(data_float);
+
+        output = format!("{}", tensor_float);
+        expected = "Tensor {\n  data: [[1.1, 2.2, 3.3], [4.4, 5.5, 6.6], [7.7, 8.8, 9.9]],\n  shape:   [3, 3],\n  device:  Cpu,\n  backend: \"ndarray\",\n  dtype:   \"float\",\n}";
+        assert_eq!(output, expected);
+
+        let data_bool = Data::from([[true, false, true], [false, true, false], [false, true, true]]);
+        let tensor_bool: burn_tensor::Tensor<TestBackend, 2, burn_tensor::Bool> =
+            Tensor::from_data(data_bool);
+
+        output = format!("{}", tensor_bool);
+        expected = "Tensor {\n  data: [[true, false, true], [false, true, false], [false, true, true]],\n  shape:   [3, 3],\n  device:  Cpu,\n  backend: \"ndarray\",\n  dtype:   \"bool\",\n}";
+        assert_eq!(output, expected);
```

Comment:
I think it's a good idea to have tests for all tensor kind even if their implementation are the same, at least for the basic 2d case. However, I would put each test in its own function like `test_display_2d_float_tensor`.

Reply:
Fixed!